packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -156,7 +156,7 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
           aria-expanded={ isOpen ? 'true' : 'false' }
           role='menu'
           aria-live={ disabled || isOpen ? 'off' : 'assertive' }
-          aria-label={ ariaLabel || label }
+          aria-label={ ariaLabel && !label ? ariaLabel : null }
```

Comment:
This seems unnecessary, we should just keep it `aria-label={ ariaLabel }`. To clarify, if user provides both `label` and `ariaLabel`, the `ariaLabel` should win and setting it as is it will take of that.